pkg/persistence/config/writer/config_writer_test.go

```diff
@@ -1023,6 +1026,47 @@ func TestWriteConfigs(t *testing.T) {
 				"project/bucket/mybucket.json",
 			},
 		},
+		{
+			name:    "Grail file-segment",
```

Comment:
```suggestion
			name:    "Grail filter segment",
```

Reply:
```suggestion
			name:    "Filter-Segments",
```

Reply:
I would keep Grail as it is in the documentation under Grail. Also the name of the Open API spec is Grail Filter-Segment